mls-rs-uniffi/src/lib.rs

```diff
@@ -262,11 +205,8 @@ pub enum ReceivedMessage {
     GroupInfo,
     /// Validated welcome message.
     Welcome,
-
-    // TODO(mgeisler): rename to `KeyPackage` when
-    // https://github.com/awslabs/mls-rs/issues/98 is fixed.
     /// Validated key package.
-    ValidatedKeyPackage { key_package: Arc<KeyPackage> },
```

Comment:
We might add `signing_identity` / `credential` here to allow verifying that we received key package for the client we actually wanted to add.

Reply:
Yes, would you like me to do this before we remove `KeyPackage` or can we do it down the road?

Reply:
Just leaving a note, doesn't have to be now. Approved the PR.

Reply:
Sounds good!